src/MOI_wrapper.jl

```diff
@@ -76,6 +78,7 @@ end
 function MOI.empty!(instance::Optimizer)
     instance.maxsense = false
     instance.data = nothing # It should already be nothing except if an error is thrown inside copy_to
+    instance.sol = Solution()
 end
 
 MOIU.needs_allocate_load(instance::Optimizer) = true
```

Comment:
This should be changed to `MOIU.supports_allocate_load(::Optimizer, copy_names::Bool) = !copy_names`
See https://github.com/JuliaOpt/SCS.jl/pull/127